docker-compose-develop.yml

```diff
@@ -24,12 +23,14 @@ services:
       S3_BUCKET: <key>
     volumes:
       - ./app:/opt/gfw-forms-api/app
+      - ./nodemon.json:/opt/gfw-forms-api/nodemon.json # Change NAME to the value of $NAME in the Dockerfile
+    command: yarn dev
     depends_on:
       - mongo
     networks:
       - gfw-forms-network
       - gfw-network
-    command: node node_modules/.bin/grunt --gruntfile app/Gruntfile.js
+    #command: node node_modules/.bin/grunt --gruntfile app/Gruntfile.js
```

Comment:
I would personally this comment, we can always look through the git history if we need it back!